competition/track2/train/train.py

```diff
@@ -20,8 +35,15 @@ def train(input_path, output_path):
     import shutil
 
     d3rlpy.seed(313)
-    n_steps = 2
-    n_steps_per_epoch = 2
+
+    # Get config parameters.
+    train_config = merge_config(
+        self=_DEFAULT_TRAIN_CONFIG,
+        other=load_config(Path(__file__).absolute().parents[0]/'config.yaml'),
```

Comment:
Semantically the same
```suggestion
        other=load_config(Path(__file__).absolute().parent/"config.yaml"),
```